operators/operating-pinot/minion-merge-rollup-task.md

```diff
@@ -1,38 +1,41 @@
 # Minion merge rollup task
 
-The Minion merge/rollup task allows a user to merge small segments into larger ones. This aggregates data at a courser granularity to improve query performance by reducing the amount of data that must be processed during query execution. This means that Pinot can benefit from improved disk storage and the query performance.
+The Minion merge rollup task lets you merge small segments into larger ones. Helps to improve query performance and disk storage by aggregating data at a courser granularity to reduce the data processed during query execution. 
 
-Currently, we support OFFLINE table APPEND use cases and REALTIME table without upsert or dedup use cases
+This task is supported for the following use cases:
+- OFFLINE tables APPEND only
+- REALTIME tables without upsert or dedup
```

Comment:
Do we want commas after "tables" in both entries here? I can see it going either way, but it might be helpful to set off the table use cases from the limitations.

Reply:
added! I thought about this too, they don't hurt :-)